### PR TITLE
feat: Set up GP2 back-end

### PR DESCRIPTION
**Pull request by peterstarling**
Rename the asap-server app to crn-server and frontend to crn-frontend. This will then enable us to name the upcoming new apps using a common pattern

### CODE REVIEW

File: .github/workflows/algolia-sync-v2.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
       - name: Build
-        run: yarn exec "yarn workspaces foreach -vptR --from '{@asap-hub/asap-cli,@asap-hub/asap-server}' run build:babel"
+        run: yarn exec "yarn workspaces foreach -vptR --from '{@asap-hub/asap-cli,@asap-hub/crn-server}' run build:babel"
```
Comment (marot) [inline .github/workflows/algolia-sync-v2.yml:52]: What about renaming asap-cli? Is it specific to crn?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "release:frontend": "yarn workspace @asap-hub/frontend release:frontend",
     "sls": "babel-node --extensions=.js,.jsx,.ts,.tsx --config-file=./babel-cjs.config.js serverless/link-plugins.ts && TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' yarn pnpify serverless",
     "start": "yarn workspaces foreach -vpi -j 1000 run start",
-    "start:backend": "yarn workspace @asap-hub/asap-server start",
+    "start:backend": "yarn workspace @asap-hub/crn-server start",
     "start:frontend": "yarn workspace @asap-hub/frontend start",
```
Comment (marot) [inline package.json:36]: crn-frontend

File: package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "release:frontend": "yarn workspace @asap-hub/frontend release:frontend",
```
Comment (marot) [inline package.json:32]: crn-frontend?

**Review by marot — COMMENTED** [review]
Nothing suspicious was found. However, I only did a code review - should I test the changes too?

File: .github/workflows/algolia-sync-v2.yml (hunk 1 above)
Comment (peterstarling) [reply .github/workflows/algolia-sync-v2.yml:52]: its not

**Review by niezgoda — COMMENTED** [review]
I would go with naming
/apps/crn/frontend instead of /apps/crn-frontend

**Review by niezgoda — CHANGES_REQUESTED** [review]
I would go with naming
/apps/crn/frontend instead of /apps/crn-frontend

**Comment by jhonny-m** [comment]
@niezgoda why is that?

**Comment by niezgoda** [comment]
@jhonny-m - to easily divide the app between crn and gp2.
It would be just easier to go to particular project and do some search in just one project :)

**Comment by jhonny-m** [comment]
if thats the reason it shouldnt really matter too much as we can just filter just as well.
I just want to keep frontend and server the same

**Comment by niezgoda** [comment]
@kmaid what do you think about the naming convention.
Should it be /apps/crn-server or /apps/crn/server ?

**Review by niezgoda — APPROVED** [review]

**Pull request merged**